contracts/bridges/BaseBridgeReceiver.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../ITimelock.sol";
+
+contract BaseBridgeReceiver {
```

Comment:
Would be nice to add some natspecs to these contracts. You can also just leave a todo for us to fill out later